gui-agent/vmside.c

```diff
@@ -2255,6 +2261,7 @@ int main(int argc, char **argv)
     int wait_fds[2];
 
     parse_args(&g, argc, argv);
+    signal(SIGCHLD, handle_sigchld);
```

Comment:
Using `signal` to set a signal handler is deprecated.  Use `sigaction`.

Reply:
Should we do anything about the use of `signal` for `SIGTERM`?

Reply:
That should be fixed too.